docs/standard/security/cross-platform-cryptography.md

```diff
@@ -20,33 +20,57 @@ This article assumes you have a working familiarity with cryptography in .NET. F
+Authenticated encryption (AE) support is provided for AES-CCM, AES-GCM, and ChaCha20Poly1305 via the <xref:System.Security.Cryptography.AesCcm?displayProperty=fullName>, <xref:System.Security.Cryptography.AesGcm?displayProperty=fullName>, and <xref:System.Security.Cryptography.ChaCha20Poly1305?displayProperty=fullName> classes, respectively.
+
+As authentication encryption requires newer platform APIs to support the algorithm, support may not be present on all platforms. The `IsSupported` static property on the classes for the algorithm can be used to detect at runtime if the current platform supports the algorithm or not.
 
-On Windows and Linux, the implementations of AES-CCM and AES-GCM are provided by the OS libraries.
+| Cipher + Mode     | Windows                 | Linux          | macOS   | iOS | Android       | Browser |
+|-------------------|-------------------------|----------------|---------|-----|---------------|---------|
+| AES-GCM           | ✔️                      | ✔️            | ⚠️      | ❌  | ✔️            | ❌      |
+| AES-CCM           | ✔️                      | ✔️            | ⚠️      | ❌  | ✔️            | ❌      |
+| ChaCha20Poly1305  | Windows 10 Build 20142+ | OpenSSL 1.1.0+ | ⚠️      | ❌  | API Level 28+ | ❌      |
 
-### AES-CCM and AES-GCM on macOS
+### AES-CCM on macOS
 
-On macOS, the system libraries don't support AES-CCM or AES-GCM for third-party code, so the <xref:System.Security.Cryptography.AesCcm> and <xref:System.Security.Cryptography.AesGcm> classes use OpenSSL for support. Users on macOS need to obtain an appropriate copy of OpenSSL (libcrypto) for these types to function, and it must be in a path that the system would load a library from by default. We recommend that you install OpenSSL from a package manager such as Homebrew.
+On macOS, the system libraries don't support AES-CCM for third-party code, so the <xref:System.Security.Cryptography.AesCcm> class uses OpenSSL for support. Users on macOS need to obtain an appropriate copy of OpenSSL (libcrypto) for this type to function, and it must be in a path that the system would load a library from by default. We recommend that you install OpenSSL from a package manager such as Homebrew.
 
 The `libcrypto.0.9.7.dylib` and `libcrypto.0.9.8.dylib` libraries included in macOS are from earlier versions of OpenSSL and will not be used. The `libcrypto.35.dylib`, `libcrypto.41.dylib`, and `libcrypto.42.dylib` libraries are from LibreSSL and will not be used.
 
+### AES-GCM and ChaCha20Poly1305 on macOS
+
+MacOS did not support AES-GCM or ChaCha20Poly1305 until macOS 10.15 for third-party code. Prior to .NET 8, <xref:System.Security.Cryptography.AesGcm> and <xref:System.Security.Cryptography.ChaCha20Poly1305> have the same requirement as AES-CCM and users must install OpenSSL for these types to function.
```

Comment:
Is a capital M here correct/consistent with Apple's OS branding?

Reply:
I think it should always be a lowercase M.